contracts/Proxy.sol

```diff
@@ -17,19 +18,35 @@ contract Proxy is Proxied {
     }
 
     /// @dev Fallback function forwards all transactions and returns all received return data.
-    function() external payable {
+    function _fallback() internal {
         address _masterCopy = masterCopy;
         assembly {
-            calldatacopy(0, 0, calldatasize)
-            let success := delegatecall(not(0), _masterCopy, 0, calldatasize, 0, 0)
-            returndatacopy(0, 0, returndatasize)
+            calldatacopy(0, 0, calldatasize())
+            let success := delegatecall(not(0), _masterCopy, 0, calldatasize(), 0, 0)
+            returndatacopy(0, 0, returndatasize())
             switch success
                 case 0 {
-                    revert(0, returndatasize)
+                    revert(0, returndatasize())
                 }
                 default {
-                    return(0, returndatasize)
+                    return(0, returndatasize())
                 }
         }
     }
+
+    /**
+     * @dev Fallback function that delegates calls to the masterCopy. 
+     * Runs when no other function in the contract matches the call data.
+     */
```

Comment:
ubernit: There is a mix of doc-comment styles in this file, both `///` and `/** */` are used.

Reply:
Thanks, I'll fix this ASAP. Plan to merge and publish on Monday. Please feel free to approve, you'd be the first I think.

Reply:
> Please feel free to approve

Already have :wink: